app/src/docs/v1.api-spec.yaml

```diff
@@ -1182,6 +1245,52 @@ paths:
           $ref: "#/components/responses/Forbidden"
         default:
           $ref: "#/components/responses/Error"
+  /sync:
+    get:
+      summary: Synchronize with the default bucket
+      description: |-
+        Synchronize the contents of the default bucket with COMS so that it can track and manage objects
+        that are already in the bucket. This avoids the need to re-upload preexisting files through the
+        COMS API.
+
+        It looks at all objects in the bucket and adds them to an internal processing queue;
+        once it is eventually processed, it is tracked by COMS.
+      operationId: sync
+      tags:
+        - Sync
+      responses:
+        "202":
+          $ref: "#/components/responses/AddedQueueLength"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        default:
+          $ref: "#/components/responses/Error"
+  /sync/status:
+    get:
+      summary: Get sync queue size
+      description: >-
+        Returns the number of objects currently in the queue
+        that are awaiting processing by COMS.
```

Comment:
Second line is not necessary. This can be collapsed into a one liner:
```yaml
description: Returns the number of objects currently in the queue
```